java/memory/src/test/java/org/apache/arrow/memory/TestBaseAllocator.java

```diff
@@ -17,14 +17,12 @@
 
 package org.apache.arrow.memory;
 
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertNotEquals;
-import static org.junit.Assert.assertNotNull;
-import static org.junit.Assert.assertTrue;
-import static org.junit.Assert.fail;
+
+import static org.junit.Assert.*;
```

Comment:
Please expand this to avoid style errors.

Reply:
OK but It's weird that I didn't see any style errors on this.

Reply:
Done